src/main.js

```diff
@@ -8,12 +8,12 @@ const configs = {
   },
   wifi: {
     network: 'wifi',
-    wifiSsid: 'resin_io',
-    wifiKey: 'the spoon jumped over the moon'
+    wifiSsid: '<wifi ssid>',
```

Comment:
Looks like its time to add `WIFI_SSID` and `WIFI_KEY` to `env.list`, and reference them as `process.env` here.